fuzzing/tools/dict_validator_test.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""
+Tests for dict_validator.py
+"""
+
+from string import hexdigits
+from dict_validator import validate_line
+import sys
+
+if __name__ == '__main__':
+    with open("dict_data/valid.dict", 'r') as dic:
```

Comment:
The file is already added to `data` attribute in `BUILD`. However, when running `bazel test`, the test failed with `FileNotFoundError`. Under the `runfiles` directory, there did exist those files. 

I also tried `os.path.join` and `$TEST_SRCDIR` to get the absolute path like [this envoy python test](https://github.com/envoyproxy/envoy/blob/defea7ecbf6f3ebffe8b9d41afa7f62322756f05/api/tools/generate_listeners_test.py#L8), but didn't work. I am wondering how to enable the `py_test` to find the data files? 

The same error for the `invalid.dict` below.

Reply:
Couple questions, what was the `$TEST_SRCDIR` environment variable? What is the value of `os.path.dirname(__file__)`? I think getting runfiles is a little tricky with Bazel and python....

Reply:
In Envoy for C++ we use a bazel runfiles library, there seems to be one for Python as well
https://github.com/bazelbuild/bazel/blob/4e4c77ae96955138006af15664074123d6a4f51d/tools/python/runfiles/runfiles.py#L16

Reply:
Thank you Asra!

I tried to `print` out the `$TEST_SRCDIR` variable and `dirname(__file__)`, but can't see the output in the terminal, neither in the `test.log`. Then I tried the `runfiles.Rlocation` in the link, it helped the bazel test find the test data!

PTAL!